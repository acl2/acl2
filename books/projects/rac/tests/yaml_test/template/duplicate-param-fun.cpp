// RAC begin

template <int a, int a>
int foo() { return 0; }
