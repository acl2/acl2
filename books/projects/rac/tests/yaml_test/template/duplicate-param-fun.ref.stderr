duplicate-param-fun.cpp:3 (17-22): Duplicate identifier declaration (could be an enum, a variable template parameter or function) `a`
 3 | template <int a, int a>
   |                  ^^^^^
note: first defined here
 3 | template <int a, int a>
   |           ^^^^^
